Project.toml

```diff
@@ -72,3 +72,10 @@ StatsBase = "0.34"
 SurfaceFluxes = "0.12.3, 0.13"
 Thermodynamics = "0.14.1, 0.15"
 julia = "1.10"
+JuliaFormatter = "2.1.7"
```

Comment:
I don't recall why exactly, but we still use JuliaFormatter 1. (not 2.) 

JuliaFormatter shouldn't be a dependency of ClimaLand, you can keep it in your default environment 

do: 
]add JuliaFormatter@1

this will install the latest v1 

to reset this to main, do: 
git checkout origin/main Project.toml